src/WingetCreateTests/WingetCreateTests/WingetCreateTests.csproj

```diff
@@ -108,6 +111,9 @@
     <None Update="Resources\TestPublisher.MismatchedMsixInExistingBundle.yaml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+	<None Update="Resources\TestPublisher.MoveInstallerFieldsToRoot.yaml">
+	  <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+	</None>

```

Comment:
nit: alignment

Reply:
Apparently, VS wasn't showing me this alignment difference (I'm sure of my eyes 😄). Deleted and re-added the file so that the line was auto-generated. Confirmed that the alignment is correct in another editor